nginx-config/nijobs.conf

```diff
@@ -0,0 +1,37 @@
+server {
+        # listen on port 80
+        listen 80;
+        # save logs here
+        access_log /var/log/nginx/access.log main;
+
+        # where the root here
+        root /usr/share/nginx/html;
+        # what file to server as index
+        index index.html index.htm;
+
+        location / {
+            # First attempt to serve request as file, then
+            # as directory, then fall back to redirecting to index.html
+            try_files $uri $uri/ /index.html;
+        }
+
+        # Media: images, icons, video, audio, HTC
+        location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
+          expires 1M;
+          access_log off;
+          add_header Cache-Control "public";
+        }
```

Comment:
Is this necessary? Can't we let nginx/the browser decide defaults?

Reply:
This ensures consistency and a good experience for everyone. I don't know the default expiry date for images, but considering that the only images we'll have are either static (our logo, some helper svgs, ...) or almost static (companies profile logos - and even these will have an appended id, making sure that if they are updated, they change) I think that 1Month is a good value.

Regarding the Cache-Control: public - it's actually better since it will allow us to cache these images even with https, if it were private, which is the default, the images wouldn't be cached

This is a really good doc on web caches: https://www.mnot.net/cache_docs/

Reply:
Sounds good then!